components/automate-backend-elasticsidecar/habitat/bin/elastic_sidecar.rb

```diff
@@ -8,7 +8,6 @@
 require 'toml-rb'
 require 'yaml'
 require 'mixlib/shellout'
-require "chefstyle"
```

Comment:
Why are we removing this. It looks like the style validator as per https://github.com/chef/chefstyle

Reply:
As we were not doing any style lintment as per chef-style, that is why we have removed that gem

Reply:
Is it something agreed upon with other Ruby developers?

Reply:
@rishichawda Can you have a look at this ?

Reply:
Long term we should be linting everything Ruby we produce as a org. All projects in the org *except* Automate are doing that.

Reply:
Sure will get it done, we have created ticket for the same https://app.zenhub.com/workspaces/the-delta-quadrant-5fc7c4c3922dc10021323fa6/issues/chef/automate/6076

Reply:
Yep. chefstyle is used for all chef oss projects so we should probably keep it.

Reply:
I have added chefstyle